### PR TITLE
TicknCook overview and logo on apps page.

### PR DESCRIPTION
**Pull request by janainawittner**
@antmonnier, ready for review.

### CODE REVIEW

File: content/apps/tickncook/en/overview.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+It works on any touch screen, TicknCook is above all ultra connected, easy to use, and offers a simple online installation.
+Available in English and French, a demo mode allows you to test TicknCook without technical constraints or need for a commercial agreement.
+
+## Integration Features
+
+Connecting TicknCook to HubRise allows to:
+
+- Create an account on TicknCook.
+- Pull orders to TicknCook.
+- Pushes the status of the whole order of an item.
```
Comment (janainawittner) [inline content/apps/tickncook/en/overview.md:34]: @antmonnier: Definitely not sure if this is true

File: content/apps/tickncook/en/overview.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  description: TicknCook overview and why restaurants should connect it to HubRise. With a connection to HubRise orders are sent to the TicknCook production screen.
+gallery:
+  - 002-en-tickncook-counter.jpg
+  - 003-en-tickncook-kitchen.jpg
+path_override: /
+app_info:
+  category: Kitchen Display Screen (KDS)
+  availability: French and English
+  website: https://tickncook.com/en/
+  contact: hello@tickncook.com
```
Comment (janainawittner) [inline content/apps/tickncook/en/overview.md:16]: These labels are not being translated into FR. 
Same for "Feedback sur cette page". The word feedback should be translated to Retour or the like.

**Review by janainawittner — COMMENTED** [review]
@antmonniern check these two points particularly.

**Comment by janainawittner** [comment]
@antmonnier, I am going to work on the **Connection** page too, so unassigned you from this for now.

**Comment by janainawittner** [comment]
@antmonnier, ready to deploy

**Comment by janainawittner** [comment]
@antmonnier, Tick'N'Cook should be changed to **TicknCook** in the Apps EN & FR apps page too. Shall I leave you to it when working on this review?

File: content/apps/tickncook/en/overview.md (hunk 2 above)
Comment (antmonnier) [reply content/apps/tickncook/en/overview.md:16]: Tracked in a separate issue: https://github.com/HubRise/website/issues/186

**Pull request merged**